mp/src/game/client/momentum/ui/MapSelection/BaseMapsPage.cpp

```diff
@@ -92,6 +92,7 @@ CBaseMapsPage::CBaseMapsPage(vgui::Panel *parent, const char *name) : PropertyPa
     SetSize(pWide, pTall);
 
     m_hFont = INVALID_FONT;
+    m_iStartMapWhenReady = -1;
```

Comment:
Since every map ID is an unsigned int and starts at 1 and goes up, you can safely make this a uint32 and default to 0 to mean "no map"